tests/spec/shell/mv.spec.js

```diff
@@ -0,0 +1,342 @@
+var Filer = require('../../..');
+var util = require('../../lib/test-utils.js');
+var expect = require('chai').expect;
+
+describe('FileSystemShell.mv', function() {
```

Comment:
I forgot, but this also needs a test for failure on any of `destination`'s parent directories not existing.